zeebe/engine/pom.xml

```diff
@@ -125,6 +120,12 @@
       <artifactId>micrometer-commons</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>net.jcip</groupId>
+      <artifactId>jcip-annotations</artifactId>
+      <scope>provided</scope>
```

Comment:
:question: why the scope is provided?

Reply:
The annotations are not retained at runtime, so there's no need to have the JAR packaged with the project.